src/app/zap-templates/zcl/data-model/chip/boolean-sensor-configuration-cluster.xml

```diff
@@ -31,12 +31,10 @@ limitations under the License.
     <field name="Audible" mask="0x2"/>
   </bitmap>
 
-  <enum name="SensitivityEnum" type="enum8">
+  <bitmap name="SensorFaultBitmap" type="bitmap16">
     <cluster code="0x0080"/>
-    <item name="High" value="0x0"/>
-    <item name="Standard" value="0x1"/>
-    <item name="Low" value="0x2"/>
-  </enum>
+    <field name="GneralFault" mask="0x1"/>
+  </bitmap>
 
   <cluster apiMaturity="provisional">
     <name>Boolean Sensor Configuration</name>
```

Comment:
In my spec copy the name of this is `Boolean State Configuration Cluster`.

We should either match the name here or update the spec to use this name.